SUI/SUI/Views/FiltersView.swift

```diff
@@ -0,0 +1,203 @@
+                                width: 50,
+                                height: 50
+                            )
+                            .foregroundColor(.orange)
+                    }
+                }
+            }
+        }
+    }
+
+    private var topGradientView: some View {
+        VStack {
+            Rectangle().toolbar {
+                ToolbarItem {
+                    Text(Constants.navigationTitleText)
+                        .foregroundColor(.white)
+                        .font(.system(size: 24, weight: .bold))
+                        .offset(x: -130)
+                }
+            }
+            .navigationBarTitleDisplayMode(.inline)
+            .opacity(0)
+            .navigationBarItems(
+                leading:
+                Button(
+                    action: {
+                        self.presentation.wrappedValue.dismiss()
+                    },
+                    label: {
+                        Image(systemName: Constants.chevronBackwardImageName)
+                            .resizable()
+                            .frame(
+                                width: 15,
+                                height: 27
+                            )
+                            .foregroundColor(.white)
+                    }
+                )
+            )
+            Image(Constants.filterImageName)
+                .resizable()
+                .frame(
+                    width: 30,
+                    height: 30
+                )
+                .foregroundColor(.white)
+                .padding(.leading, 300)
+                .padding(.top, -45)
+        }
+        .frame(
+            width: UIScreen.main.bounds.width,
+            height: 80
+        )
+        .background(
+            LinearGradient(
+                colors: [.yellow, .orange],
+                startPoint: .leading,
+                endPoint: .trailing
+            )
+        )
+        .ignoresSafeArea()
+    }
+
+    private var categoryStackView: some View {
+        HStack {
+            Text(Constants.categoryText)
+                .font(.system(size: 25, weight: .bold))
+            Spacer()
+            Text(Constants.moreText)
+                .font(.system(size: 25, weight: .bold))
+                .foregroundColor(.gray)
+            Image(systemName: Constants.chervonForwardImageName)
+                .resizable()
+                .frame(
+                    width: 15,
+                    height: 20
+                )
+                .foregroundColor(.gray)
+        }
+        .padding(.top)
+    }
+
+    private var colorTextView: some View {
+        Text(Constants.colorText)
+            .font(.largeTitle)
+            .bold()
+            .foregroundColor(viewModel.textColor)
+            .padding(.leading, -180)
+    }
+
+    private var sliderView: some View {
+        VStack {
+            Slider(
+                value: $viewModel.sliderValue,
+                in: 100 ... 5000,
```

Comment:
```suggestion
                in: 100 ... 5000,
```
вынеси в макс мин переменные